docs/guides/mobile-plugin-gma-android.md

```diff
@@ -29,9 +29,13 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-- [Google Mobile Ads SDK v22.0.0+](https://developers.google.com/admob/android/rel-notes)
-- [UID2 SDK for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk) (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
-- [UID2 Android GMA Plugin v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-gma/)
+1. Google Mobile Ads SDK v22.0.0 or later:
+   - [SDK](https://developers.google.com/admob/android/sdk)
+   - [Release notes](https://developers.google.com/admob/android/rel-notes)
+1. UID2 SDK for Android v0.5.0:
+   - [SDK](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk)
+   - [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
+1. [UID2 Android GMA Plugin v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-gma/)
```

Comment:
The link should be in a bullet point under the name to be consistent with the other two.  Ex:

```
UID2 Android GMA Plugin v0.5.0 or later
  - [SDK](https://central.sonatype.com/artifact/com.uid2/uid2-)
```

Reply:
@bleege the others are only lists because there is more than one. I did try it but it doesn't look right. LMK if you feel strongly about this.

Reply:
Sounds good.  It's not anything that big.